Odata-docs/odatacli/getting-started.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "OData CLI"
+description: "How to generate code using the odata-cli dotnet tool"
+
+author: ElizabethOkerio
+ms.author: elokerio
+ms.date: 3/30/2022
+ms.topic: article
+ 
+---
+# OData CLI
+
+**Applies To**: [!INCLUDE[appliesto-odataclient](../includes/appliesto-odataclient-v7.md)]
+
+## Install
+There are 2 ways to download and install the **OData CLI**
+
+ - dotnet global tool:
+ 
+    To install the latest release of the OData CLI [Nuget Package](https://www.nuget.org/packages/Microsoft.OData.Cli/), use the [dotnet tool install](/dotnet/core/tools/dotnet-tool-install) command
+
+    ```.NET CLI
+    dotnet tool install --global Microsoft.OData.Cli --version 0.1.0
+    ```
+
+ - Direct Download:
+
+    You can download the binary files from this [GitHub Release](https://github.com/OData/ODataConnectedService/releases/tag/v0.1.0) and use them directly to generate code.
+
+## Synopsis
+
+```Console
+odata-cli [-h|--help] [--version] <command>
+```
+
+## Description
+
+**odata-cli** is a CLI tool that generates strongly-typed C# and Visual Basic client code for a specified OData service. It generates a [DataServiceContext](/dotnet/api/microsoft.odata.client.dataservicecontext) class to interact with the service and CLR types for each entity type and complex type in the service model.
+
+## Options
+
+ - `version`
+
+    Displays the version of the odata-cli
+ - `-h|--help`
+
+    Shows the command-line help
+
+
+## Commands
+
+### generate
+
+It generates strongly typed C# and Visual Basic client code for a specified OData service. 
+
+### Synopsis
+
+```Console
+odata-cli generate [-h|--help] [-m|--metadata-uri] [-fn|--file-name] [-h|--custom-headers] [-p}--proxy] [-ns|--namespace-prefix] [-ucc|--upper-camel-case] [-i|--internal] [--multiple-files] [-eoi|--excluded-operation-imports] [-ebo|--excluded-bound-operations] [-est|--excluded-schema-types] [-o|--outputdir]
+```
+
+### Options 
+
+- `--metadata-uri|-m` 
+
+     The URI of the metadata document. The value must be set to a valid service document URI or a local file path. This is a required option
+
+ - `--file-name|-fn`
```

Comment:
nit: why not have this as `--file|-f`? These seem more in line with standard convention for cli tools